packages/container/libraries/common/vitest.js.workspace.mjs

```diff
@@ -0,0 +1,3 @@
+import { jsWorkspaceConfig } from '@inversifyjs/foundation-vitest-config';
+
+export default jsWorkspaceConfig;
```

Comment:
Let's get rid of this, I honestly believe it's not worth to target compiled tests.